src/utils/theme/index.js

```diff
@@ -40,9 +40,11 @@ export const COLORS = {
 
   // new color
   signed_primary: '#4782D7',
+  signed_primary_15: '#4782D726',
   signed_secondary: '#6295DD',
   default_signed_secondary: '#69A2F2',
   anon_primary: '#107793',
+  anon_primary_15: '#10779326',
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The new color definitions `signed_primary_15`, `anon_primary_15` are introduced. It seems like these colors are a 15% opacity version of the primary colors. However, it's not clear how these colors are calculated. If they are indeed intended to be 15% opacity versions of the primary colors, it would be better to calculate them programmatically to avoid errors and improve maintainability.
```javascript
signed_primary_15: `${signed_primary}26`,
anon_primary_15: `${anon_primary}26`,
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->